cmd/kube-apiserver/app/server.go

```diff
@@ -730,6 +730,19 @@ func defaultOptions(s *options.ServerRunOptions) error {
 		}
 	}
 
+	// TODO: remove when we stop supporting the legacy group version.
+	if s.APIEnablement.RuntimeConfig != nil {
+		for key, value := range s.APIEnablement.RuntimeConfig {
+			if key == "v1" || strings.HasPrefix(key, "v1/") ||
+				key == "api/v1" || strings.HasPrefix(key, "api/v1/") {
```

Comment:
This can not enable disable resources in core v1

Reply:
see https://github.com/kubernetes/kubernetes/pull/62254/commits/2a9ebb3f6678fe70348f23fa40f5c6b234078a3e#diff-3d1aa7e5d43db965581b230647a90de2R71